incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -257,6 +257,11 @@ func RecoverHNC() {
 	}
 	// give HNC enough time to repair
 	time.Sleep(5 * time.Second)
+	// verify
```

Comment:
Can you please add a much longer comment here about what it is you're trying to do? (But good solution!)

Reply:
done